.circleci/config.yml

```diff
@@ -1,6 +1,19 @@
 version: 2.1
 
 jobs:
```

Comment:
Could you add a comment in this file explaining that it intends to do the same thing that .travis.yml currently does and that there is a plan to replace Travis with CircleCI? (ideally file an issue tracking that replacement if there isn't one yet, and mention it here and in the PR description)

Reply:
It's broken out into a bunch of tickets!